Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/StatisticsHomeCard.kt

```diff
@@ -47,10 +47,25 @@ class StatisticsHomeCard(
         }.let { statsAdapter.update(it) }
     }
 
+    @Suppress("EqualsOrHashCode") // ignore warning: custom implementation of hashCode() not needed
     data class Item(
         val data: StatisticsData,
         val onHelpAction: (StatsItem) -> Unit
     ) : HomeItem {
         override val stableId: Long = Item::class.java.name.hashCode().toLong()
+
+        // ignore onHelpAction so that view is not re-drawn when only the onHelpAction click listener is updated
+        // Auto-generated
+        override fun equals(other: Any?): Boolean {
```

Comment:
This works, but if you override `equals` please also also override `hashcode`.

Reply:
Done.